routes/index.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Set up routes for save/publish
+*/
+
+module.exports = function(server) {
+
+  // /projects/* API routes
+  // Handles user-level operations
+  require('./projects/get-projects.js')(server);
+  require('./projects/create-project.js')(server);
+  require('./projects/delete-project.js')(server);
+  require('./projects/publish-project.js')(server);
+
+  // /files/* API routes
+  // Handles project-level operations
+  require('./files/get-project-contents.js')(server);
+  require('./files/get-file-contents.js')(server);
```

Comment:
I wonder if you should split out `project/` and `file/` so you can get project contents or file contents?  So you'd have routes for:
* `/projects/...`
* `/project/...`
* `/file/...`